Core/src/Geometry/TrackingGeometry.cpp

```diff
@@ -53,7 +53,12 @@ const Acts::TrackingVolume* Acts::TrackingGeometry::lowestTrackingVolume(
 
 const Acts::TrackingVolume* Acts::TrackingGeometry::highestTrackingVolume()
     const {
-  return (m_world.get());
+  return m_world.get();
+}
+
+std::shared_ptr<const Acts::TrackingVolume>
```

Comment:
Probably unnecessary but

```suggestion
const std::shared_ptr<const Acts::TrackingVolume>&
```